src/datetime.rs

```diff
@@ -1288,6 +1288,13 @@ pub mod serde {
             {
                 Ok(None)
             }
+
+            /// Deserialize a timestamp in seconds since the epoch
+            fn visit_unit<E>(self) -> Result<Option<DateTime<Utc>>, E>
+                where E: de::Error
+            {
+                Ok(None)
+            }
```

Comment:
@quodlibetor, regarding your comment about reproducibility:
Your test actually works as intended.
Removing this method results in the following error for me (enabling the `serde` feature):
```
$ cargo test --features serde
# [...]
test src/datetime.rs - datetime::serde::ts_milliseconds_option::deserialize (line 1537) ... FAILED
# [...]
failures:

---- src/datetime.rs - datetime::serde::ts_milliseconds_option::deserialize (line 1537) stdout ----
Test executable failed (exit code 101).

stderr:
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `time`", line: 1, column: 2)', src/datetime.rs:26:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace



failures:
    src/datetime.rs - datetime::serde::ts_milliseconds_option::deserialize (line 1537)

test result: FAILED. 243 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out
```

Reply:
Oh, it even fails without change, let me have a deeper look into this. Sorry!